cmd/tester/server.go

```diff
@@ -106,9 +108,8 @@ func (s *Server) Run() error {
 	http.HandleFunc("/start", s.startHandler)
 	http.HandleFunc("/stop", s.stopHandler)
 	http.HandleFunc("/exit", s.exitHandler)
-	http.ListenAndServe(s.WebAddr, nil)
 
-	return nil
+	return http.ListenAndServe(s.WebAddr, nil)
```

Comment:
We were ignoring the error returned from the `ListenAndServe`, is this a worthwhile change?

Reply:
Try to keep unrelated changes to a minimum. The tester is also a tester, and doesn't need the same level of care.

Reply:
Cool, I have reverted it back.